### ISSUE TITLE
Feature request: add an option to exclude Z<> comments from the input pod

### ISSUE THREAD

**Issue by tbrowder** [JJ/p6-pod-load#15]
By my reading of Raku pod documentation I should be able to write pod comments that will not be rendered. Since, in effect, Pod::Load is a renderer, it would be nice to have an option to exclude pod comments from the rendered (i.e., loaded) pod.

I'll submit a PR for this feature.

**Comment by tbrowder** [comment]
I'm sorry I started the original issue and this "feature request." I misunderstood the real purpose of the module which does exactly what it is intended to do. Interpretation of the pod is the job of the user. Merry Christmas, @JJ!

**Comment by JJ** [comment]
Hope the new feature helps you

**Comment by tbrowder** [comment]
I am revisiting pod handling and remember one of the problems in my simple mind is there is no "ending" pod type to stop possibly infinite recursion due to the @contents attribute. I proposed a Pod type which would ease using programs' analysis of text as well as ease handling of formatting code in table cells.

I got shot down, but maybe I should look at implementing a module to accomplish that instead,

I have solved my immediate problem by trying to strictly definine pod that is acceptable and expected, and that throws with a helpful msg if something is unacceptable or unrecognized.
